### ISSUE TITLE
Book id leading to wrong books

### ISSUE THREAD

**Issue by tofuwabohu** [bookwyrm-social/bookwyrm#2098]
I read 1984 and marked it as read. It showed up in the read category. Today I found that there's a different book in it's spot.
![image](https://user-images.githubusercontent.com/66644064/166523504-7839bda5-d8a7-471b-bce1-7d2ddb273410.png)
When I click it in the shown "Your Books", the title in the posting box still says [Nineteen Eighty-Four (1984)](https://wyrms.de/book/39800) by [George Orwell](https://wyrms.de/author/5182). Clicking Orwell leads to his author page but clicking the book title it opens book 39800 which is _Outliers: The Story of Success_ by Malcolm Gladwell.

There are 3 reviews and some quotes saved to book 39800 which I think refer to 1984.

The last edit was 2 days ago by a bookwyrm.social user. I can't see what exactly they changed though, is it possible that they replaced the whole book's data with another book? The user also reviewed 1984 which makes that unlikely.

I'm not sure anymore how the editor role works. Maybe the user is an editor on b.s and since that edition was used as a source to import it and then the changes federated? It has an OL link though that leads to the new book.

[Here](https://bookwyrm.social/book/329660) is the version on b.s with a review that now shows up on the w.d Outliers book.

**Comment by mouse-reeve** [comment]
This is bizarre! It looks like this user created this edition, and I'd guess that something went wrong when the edition federated that caused it to be linked to the wrong corresponding edition on wyrms.de. It's not obvious what happened, though. On b.s, the ISBN 10 field contains an ISBN13, which is odd but doesn't really explain what happened.

**Comment by tofuwabohu** [comment]
The book is back to normal, except for the cover which is a new one. The editing page says a wyrms.de user edited it 13h ago but the user doesn't have the book in their bookshelf, of course it's possible that they edited anyways, I'll ask them.

**Comment by tofuwabohu** [comment]
It was a user who noticed and fixed. I guess it was an unlucky combination of someone trying to create a new book editing an existing one and federation? Since different titles were shown at the book page and my profile, I think there's still some bug somewhere.

**Comment by mouse-reeve** [comment]
I agree that there's a bug in it, but I'm glad for the moment it's fixed. I think the route to investigate is what happens when a federated book tries to match up with the existing database when there's insufficient metadata (especially when matching `Work`s)

**Comment by mouse-reeve** [comment]
Closing this because it's very old
